narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -15,7 +15,12 @@ function check_if_tag_exists {
 
 	if (git -P tag -l "${tag_name}" | grep -q "[[:alnum:]]")
 	then
-		lc_log DEBUG "The tag '${tag_name}' already exists in the ${repository} repository."
+		lc_log DEBUG "The tag '${tag_name}' already exists in the ${repository} repository. Updating ${IGNORE_ZIP_FILES_CACHE_FILE}."
```

Comment:
I am wondering if it would be more obvious and readable if we separate the check and action logically.
It would also make debugging potentially easier.
So if lc_log would be left as is.

And there would be another lc_log DEBUG after the if, something like:

```lc_log DEBUG "Add "${hotfix_zip_file}" to "${IGNORE_ZIP_FILES_CACHE_FILE}"```
IMO it better fits there logically. What do you think?

Reply:
Yes it looks good, I'll add it